fah_xchem/data/__init__.py

```diff
@@ -0,0 +1,16 @@
+
+from os.path import dirname, join
+import json
+import bz2
+
+from ..schema import TimestampedAnalysis
+
+def get_compound_series_analysis_results():
+    module_path = dirname(__file__)
+
+    compound_series_analysis_file = join(module_path, "results/minimal-neq-sprint/minimal-neq-sprint-stereofilter-P0033-dimer-neutral-restrained/analysis.json.bz2")
```

Comment:
Is this hard-coded path able to be overridden?

Reply:
Oh! This is just for unit testing, right?

Reply:
Indeed! This is just a function for pulling out the test data included in the package.